lib/types/beacon_chain/deposit.ex

```diff
@@ -40,4 +41,12 @@ defmodule Types.Deposit do
       slashed: false
     }
   end
+
+  @impl LambdaEthereumConsensus.Container
+  def schema do
+    [
+      {:proof, {:list, {:bytes, 32}, 33}},
```

Comment:
Please add a new issue titled "Change lists to vectors in schemas where needed". After the ssz vectors PR is merged, we need to do that change.

The PR should include all changes needed included this one (e.g. "Deposit.proof").

Reply:
since the ssz vector PR is already merged, I am going to map these kinds of types directly into `:vector`